src/openscad.cc

```diff
@@ -794,7 +794,7 @@ struct CommaSeparatedVector {
 };
 
 // OpenSCAD
-int main(int argc, char **argv)
+int openscad_main(int argc, char **argv)
```

Comment:
Super nit: If we can make this const char **argv, that would help later, but I think Python may need non-const.

Reply:
I also have to move `wmain` to `main_wrapper.cc`